src/Compiler/Reports/SynthesisReportManager.cpp

```diff
@@ -29,13 +29,22 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "TableReport.h"
 
 namespace {
+// Report strings
 static constexpr const char *REPORT_NAME{"Synthesis report"};
 static constexpr const char *MAX_LVL_STR{"Maximum logic level"};
 static constexpr const char *AVG_LVL_STR{"Average logic level"};
+
+// Messages regexp
+static const QRegExp MESSAGES_REGEXP{
```

Comment:
Just a thought, we can let user comments configure the messages to be extracted via reading a file or settings dialog.

Reply:
that might be a powerful features, yes. but sounds rather like a new feature
we may also want to support RegEx there and validate users input.

Reply:
Yes it is but keep it in mind since it adds flexibility for each type of user.